opencti-platform/opencti-front/src/private/components/profile/triggers/TriggerLine.tsx

```diff
@@ -247,11 +248,19 @@ export const TriggerLineComponent: FunctionComponent<TriggerLineProps> = ({
                 )}
               </div>
             )}
+            <div
+              className={classes.bodyItem}
+              style={{ width: dataColumns.ownership.width }}
+            >
+              {data.currentUserAccessRight === 'admin' ? t('User') : t('Group')}
```

Comment:
I don't understand this condition
If I create a trigger in Group, the owner is the user
So, if an other user is in this Group, the owner should be this main user no ?

Reply:
as discussed with @Kedae i'll change the wording for `Private` and `Group`

Reply:
Private and Shared :)

Reply:
we won't have anymore column but a grayed out burger button with a tooltip "This trigger/digest has been shared with you an you are not able to modify or delete it."